fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -596,20 +595,13 @@ public void checkTimeout(long now) {
                 killConnection = true;
             }
         } else {
-            long timeout;
             String timeoutTag = "query";
-            if (userQueryTimeout > 0) {
-                // user set query_timeout property
-                timeout = userQueryTimeout * 1000L;
-            } else {
-                //to ms
-                timeout = executionTimeoutS * 1000L;
-            }
-            //deal with insert stmt particularly
+            // insert stmt particularly
             if (executor != null && executor.isInsertStmt()) {
                 timeoutTag = "insert";
             }
-
+            //to ms
+            long timeout = executionTimeoutS * 1000L;
```

Comment:
```suggestion
            long timeout = getExecTimeout() * 1000L;
```
Why not use method to get the timeout value

Reply:
Because MysqlLoadManager#executeMySqlLoadJobFromStmt seems have to use setExecutionTimeout to directly control the kill thread behavior, if I use getter here, will it affect the MysqlLoadManager#executeMySqlLoadJobFromStmt?